scarb/src/compiler/compilers/lib.rs

```diff
@@ -127,3 +149,16 @@ impl Compiler for LibCompiler {
         Ok(())
     }
 }
+
+/// Represents a contract in the Starknet network.
+#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct SerializedCasm {
```

Comment:
Why don't we just define some serialization for the struct in Cairo repo (in that repo)? https://github.com/starkware-libs/cairo/blob/233c0fdabe8c5010d10b5d16e02fecf6b656ef6b/crates/cairo-lang-sierra-to-casm/src/compiler.rs#L95

Reply:
You're right it should ideally go there but I wasn't sure if the cairo team wanted to make this a part of that repo based on some discussion in related issues. I'll open a PR there.